scripts/linters/general_purpose_linter.py

```diff
@@ -37,7 +37,7 @@
     '*.mp3', '*.mp4', 'node_modules/*', 'typings/*', 'local_compiled_js/*',
     'webpack_bundles/*', 'core/tests/services_sources/*',
     'core/tests/release_sources/tmp_unzip.zip',
-    'core/tests/release_sources/tmp_unzip.tar.gz')
+    'core/tests/release_sources/tmp_unzip.tar.gz', 'tmpcompiledjs/*')
```

Comment:
The string tmpcompiledjs/ is scattered all over the place. I suggest making it a named constant and referring to that named constant everywhere, so that the dependencies on it can be more clearly identified.

Reply:
Done